### PR TITLE
vtctldclient OnlineDDL: support `throttle`, `unthrottle`

### PR DESCRIPTION
**Pull request by shlomi-noach**
## Description

Implementing `vtctldclient OnlineDDL throttle|unthrottle` by invoking `UpdateThrottlerConfig`. Example usage:

```sh
$ vtctldclient OnlineDDL throttle commerce all
$ vtctldclient OnlineDDL unthrottle commerce all
$ vtctldclient OnlineDDL throttle commerce b1f35e60_4be4_11ee_b92f_0a43f95f28a3
$ vtctldclient OnlineDDL unthrottle commerce b1f35e60_4be4_11ee_b92f_0a43f95f28a3
```

Throttling expires using the default expiration, which is `1h`. Whether that is a good value or not is to be determined in a different discussion.

## Related Issue(s)

- https://github.com/vitessio/vitess/issues/13712
- Fixes https://github.com/vitessio/vitess/issues/13720
- Fixes https://github.com/vitessio/vitess/issues/13721

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on the CI
-   [ ] Documentation was added or is not required

### CODE REVIEW

**Comment by shlomi-noach** [comment]
Given the implementation calls `UpdateThrottlerConfig`, I don't have a similar test story to the `Complete`, `Cancel` etc. commands. Thoughts on a good test case?

File: go/vt/vtgate/planbuilder/migration.go [hunk 1]
```diff
@@ -40,7 +41,7 @@ func validateThrottleParams(alterMigrationType sqlparser.AlterMigrationType, exp
 		// Unthrottling is like throttling with duration=0
 		duration = 0
 	default:
-		duration = time.Hour * 24 * 365 * 100
+		duration = throttle.DefaultAppThrottleDuration
```
Comment (shlomi-noach) [inline go/vt/vtgate/planbuilder/migration.go:44]: Unrelated to the PR, but I just realized this wasn't using the default expiration, and so was not inline with all other `throttle` commands.

**Review by ajm188 — APPROVED** [review]
> 
Given the implementation calls UpdateThrottlerConfig, I don't have a similar test story to the Complete, Cancel etc. commands. Thoughts on a good test case?

I think relying on the `UpdateThrottlerConfig` tests is fine for now; we don't have a good story at all for testing things from the `vtctldclient` cli end (aside from endtoend tests that invoke the client as part of other testing)

**Comment by ajm188** [comment]
Also noting that we'll need a website docs update to correspond with this

File: go/cmd/vtctldclient/command/onlineddl.go [hunk 2]
```diff
@@ -83,6 +86,22 @@ var (
 		Args:                  cobra.ExactArgs(2),
 		RunE:                  commandOnlineDDLRetry,
 	}
+	OnlineDDLThrottle = &cobra.Command{
+		Use:                   "throttle <keyspace> <uuid|all>",
+		Short:                 "throttles one or all migrations",
```
Comment (ajm188) [inline go/cmd/vtctldclient/command/onlineddl.go:91]: nit: these are full sentences with capitalization/punctiuations for some of the others

Comment (ajm188) [reply go/cmd/vtctldclient/command/onlineddl.go:91]: (see `Show` and `Cleanup`)

**Review by ajm188 — COMMENTED** [review]
couple non-blocking post-approval comments!

File: go/cmd/vtctldclient/command/onlineddl.go (hunk 2 above)
Comment (shlomi-noach) [reply go/cmd/vtctldclient/command/onlineddl.go:91]: fixed

**Comment by shlomi-noach** [comment]
Documentation PR: https://github.com/vitessio/website/pull/1577

**Review by ajm188 — APPROVED** [review]

File: go/cmd/vtctldclient/command/onlineddl.go [hunk 3]
```diff
@@ -239,6 +258,82 @@ func commandOnlineDDLRetry(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// commandOnlineDDLThrottle throttles one or multiple migrations.
+// As opposed to *most* OnlineDDL functions, this functionality does not end up calling a gRPC on tablets.
+// Instead, it updates Keyspace and SrvKeyspace entries, on which the tablets listen.
+func commandOnlineDDLThrottle(cmd *cobra.Command, args []string) error {
```
Comment (rohit-nayak-ps) [inline go/cmd/vtctldclient/command/onlineddl.go:264]: The only difference b/w the two handlers appears to the `Ratio` and `ExpiresAt` vallues in the rule. We could  abstract the rest of the code into a helper, and call it from both Throttle/Unthrottle.

**Review by rohit-nayak-ps — COMMENTED** [review]
Nice that we have OnlineDDL now fully on vtctldclient!

Left a suggestion, at your discretion.

File: go/cmd/vtctldclient/command/onlineddl.go (hunk 3 above)
Comment (shlomi-noach) [reply go/cmd/vtctldclient/command/onlineddl.go]: Done. The two functions are consolidated in a single helper function.

**Review by rohit-nayak-ps — APPROVED** [review]

**Pull request merged**